app/views/dashboard/index.html.erb

```diff
@@ -47,9 +47,9 @@
 
 
       <div class="box-header with-border bg-gray">
-        <section id="filters">
+        <section id="filters" class="submit-on-change">
           <div class="box-body">
-              <%= simple_form_for :dashboard_filter, url: dashboard_path(current_organization), remote: true, method: :get do |f| %>
+              <%= simple_form_for :filters, url: dashboard_path(current_organization), remote: true, method: :get do |f| %>
```

Comment:
I don't know what `:filters` vs `:dashboard_filter` really does -- only a name that doesn't get referenced?

Reply:
ah, gets put into the ids and is what makes `id=filters_interval` below. I see.